src/pages/MissionsList.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMissionsAction } from '../redux/missions';
+import Mission from './Mission';
+import styles from './MissionsList.module.css';
+
+const MissionsList = () => {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
```

Comment:
[ Optional ] I suggest you dispatch the getMissionsAction in the index.js file since it loads only once. This will prevent a dispatch anytime you move to the missions page.

Reply:
Thank you, I did as you said.